solutions/devsprint-eduardo-bocato-1/SpaceApp/Modules/Features/Home/Scenes/Home/HomeScene.swift

```diff
@@ -3,31 +3,53 @@ import SwiftUI
 
 struct HomeScene: View {
     @StateObject var viewModel: HomeViewModel
-    
+    @State private var tabSelected: Int = 1
+
     init(viewModel: HomeViewModel) {
         self._viewModel = .init(wrappedValue: viewModel)
     }
     
     var body: some View {
-        NavigationView {
-            List(viewModel.launches) { launch in
-                Text("\(launch.name)")
-            }
-            .onAppear { viewModel.loadLaunches() }
-            .navigationTitle("Space App 🚀")
+        TabView(selection: $tabSelected) {
+
+            NavigationSceneView(title: SpaceApp.HomeScene.homeTitle) {
+                Text(SpaceApp.HomeScene.homeTitle)
+            }.tabItem {
+                Label(SpaceApp.HomeScene.homeTitle, systemImage: "house.fill")
+            }.tag(1)
+
+            NavigationSceneView(title: SpaceApp.Common.appTitle) {
+                VStack {
+                    List(viewModel.launches) { launch in
+                        Text("\(launch.name)")
+                    }
+                    .onAppear { viewModel.loadLaunches() }
+                }
+            }.tabItem {
+                Label(SpaceApp.HomeScene.launchesTitle, systemImage: "location.north.fill")
+            }.tag(2)
```

Comment:
**[CHANGE REQUEST]**
Consider modelling your tabs as a enum instead of using a magic number...
```swift
enum HomeTab: Int {
     case launches
     case otherTab
}
```

Reply:
Nice catch, thank you!